spec/controllers/collection_draft_proposals_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe Proposal::CollectionDraftProposalsController do
+  it 'shows draft collection proposal record for proposal mode' do
+    set_as_proposal_mode_mmt
+    @collection_draft_proposal = create(:full_collection_draft_proposal, user: get_user)
+    allow_any_instance_of(Cmr::UrsClient).to receive(:validate_dmmt_token).and_return(Cmr::Response.new(Faraday::Response.new(status: 200, body: '{"uid":"testuser"}', response_headers: {})))
+    get :show, params: { id: @collection_draft_proposal.id }, as: :json
+    parsed_body = JSON.parse(response.body)
+    draft_json = @collection_draft_proposal.draft
```

Comment:
nitpicks: I'd like to see lines 3-8 in a `before` block as well as a `context` block surrounding the `before` and `it` blocks.

Reply:
Good point.